app/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,7 @@
     <string name="sign_in_description">Sign in to continue</string>
     <string name="sign_in_with_google">Sign in with Google</string>
     <string name="please_wait">Please wait...</string>
-
+    <string name="search_icon">Search Icon</string>
```

Comment:
Nit: I think this is similar to the comment in the other PR about "Close Icon, Button, double-tap to activate" announcements. Can we please check these?